modules/data.land/R/pool_ic_list2netcdf.R

```diff
@@ -0,0 +1,98 @@
+##' @name pool_ic_list2netcdf
+##' @title pool_ic_list2netcdf
+##' @export
+##'
```

Comment:
Would be helpful to provide a description summarizing what this function does (It writes a list of initial conditions into a validated netCDF, correct?)